packages/typespec-client-generator-core/test/types/body-model-property-types.test.ts

```diff
@@ -11,17 +11,6 @@ describe("typespec-client-generator-core: body model property types", () => {
     runner = await createSdkTestRunner({ emitterName: "@azure-tools/typespec-java" });
   });
 
-  afterEach(async () => {
-    for (const modelsOrEnums of [
-      runner.context.sdkPackage.models,
-      runner.context.sdkPackage.enums,
-    ]) {
-      for (const item of modelsOrEnums) {
-        ok(item.name !== "");
-      }
-    }
-  });
-
```

Comment:
why we remove such post tests? it is used to check if tcgc could provide all types with name.

Reply:
i removed them bc they do make debugging a lot messier. I'll add them back bc I don't feel too strongly